.husky/prepare-commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+exec < /dev/tty && npx cz --hook || true
```

Comment:
Does it work on all platforms ?

Reply:
i used this on mac osx, no testing on windows

Reply:
Thx, I'll try on linux to see. Would be great if it works on Windows but not required.